nix/m1-support/kernel/default.nix

```diff
@@ -11,18 +11,18 @@
     # we definitely want to use CONFIG_ENERGY_MODEL, and
     # schedutil is a prerequisite for using it
     # source: https://www.kernel.org/doc/html/latest/scheduler/sched-energy.html
-    powerManagement.cpuFreqGovernor = lib.mkOverride 800 "schedutil";
+    powerManagement.cpuFreqGovernor = lib.mkDefault "schedutil";
```

Comment:
This has to stay at `lib.mkOverride 800` to avoid conflicting with the nixos-generate-config default and any user who wants to change this.